site/docs/evolution.md

```diff
@@ -62,3 +62,40 @@ When you evolve a partition spec, the old data written with an earlier spec rema
 Iceberg uses [hidden partitioning](./partitioning.md), so you don't *need* to write queries for a specific partition layout to be fast. Instead, you can write queries that select the data you need, and Iceberg automatically prunes out files that don't contain matching data.
 
 Partition evolution is a metadata operation and does not eagerly rewrite files.
+
+Iceberg's Java table API provides `updateSpec` API to update partition spec. 
+For example, the following code could be used to update the partition spec to 
+add a new partition field that places `id` column values into 8 buckets,
+remove an existing partition field `category`, and rename a partition field `id_bucket_8` to `shard`:
```

Comment:
It looks like this is renaming a partition field that was just added because `bucket("id", 8)` will be named `id_bucket_8`. What about simplifying this by removing the rename?

Reply:
Sure, I will remove it